css/style.css

```diff
@@ -360,6 +360,18 @@ a:hover {
         left: 1px;
         content: "\f0c0";
     }
+    .vcard .githubProfile:before {
+        position:absolute;
```

Comment:
Could you add a space after the colon here? Nitpicky, but you'll want to squash your commits anyway so might as well :smile: